### ISSUE TITLE
Conflict with Shared Vision

### ISSUE THREAD

**Issue by LorduFreeman** [dev7355608/perfect-vision#98]
https://github.com/CDeenen/SharedVision

```
D: Conflict detected between module 'perfect-vision' and module 'SharedVision'.
Failed to wrap 'ForegroundLayer.prototype.updateOcclusion' for module 'perfect-vision' with type OVERRIDE. An OVERRIDE wrapper for the same method has already been registered by module 'SharedVision'.
    at Function.🎁register [as register] (<SERVERURL>/modules/lib-wrapper/lib-wrapper.js:1:30598)
    at patch (<SERVERURL>/modules/perfect-vision/scripts/utils/patch.js:80:16)
    at <SERVERURL>/modules/perfect-vision/scripts/core/masks/occlusion.js:49:5
    at Function._call (eval at <anonymous> (<SERVERURL>/modules/lib-wrapper/lib-wrapper.js:1:33931), <anonymous>:4:14)
    at Function.callAll (<SERVERURL>/scripts/foundry.js:153:12)
    at Game.initialize (<SERVERURL>/scripts/foundry.js:4190:11)
    at x.🎁call_wrapped [as call_wrapped] (<SERVERURL>/modules/lib-wrapper/lib-wrapper.js:1:15517)
    at Game.🎁libWrapperInit (<SERVERURL>/modules/lib-wrapper/lib-wrapper.js:1:35638)
    at Game.🎁Game.prototype.initialize#0 (<SERVERURL>/modules/lib-wrapper/lib-wrapper.js:1:12240)
    at window.addEventListener.once (<SERVERURL>/scripts/foundry.js:53180:26)
```

Shared Vision already uses libwrapper, would be nice if Perfect Vision would use it as well.

Foundry 0.8.8
DnD5e 1.4.1
PV 2.2.0
SharedVision 1.0.6

**Comment by dev7355608** [comment]
Duplicate of #91.

We both use libwrapper, but it's two overrides, so it has to complain. I already included in my override what shared vision is doing. If you give PV higher priority in libwrapper, everything works fine even though you get the error message. I'll need to discuss with the author of Shared Vision how to resolve this confict without the error message.

**Comment by dev7355608** [comment]
Conflict resolved in PV 2.4.0.
